testproject/Assets/ManualTests/ClientCounterBehaviour.cs

```diff
@@ -0,0 +1,616 @@
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using MLAPI;
+using MLAPI.Messaging;
+
+/// <summary>
+/// This class is used to verify that the RPC Queue allows for the
+/// sending of ClientRpcs to specific clients.  It has several "direct"
+/// methods of sending as is defined in the enum ClientRpcDirectTestingModes:
+///     Single: This will send to a single client at a time
+///     Striped: This will send to alternating client pairs at a time
+///     Unified: This sends to all clients at the same time
+/// Each direct testing mode sends a count value starting at 0 up to 100.
+/// Each direct testing mode sends to all clients at least once during testing (sometimes twice in striped)
+/// During direct testing mode the following is also tested:
+///     all clients are updating the server with a continually growing counter
+///     the server is updating all clients with a global counter
+/// During all tests the following additional Rpc Tests are performed:
+///     Send client to server no parameters and then multiple parameters
+///     Send server to client no parameters and then multiple parameters
+/// </summary>
+public class ClientCounterBehaviour : NetworkBehaviour
+{
+    private const float  k_ProgressBarDivisor = 1.0f/200.0f;
+
+    [SerializeField]
+    private Text m_CounterTextObject;
+
+    [SerializeField]
+    private Image m_ClientProgressBar;
+
+    [SerializeField]
+    private GameObject m_ConnectionModeButtonParent;
+
+    private Dictionary<ulong,int> m_ClientSpecificCounters = new Dictionary<ulong, int>();
+    private List<ulong> m_ClientIds = new List<ulong>();
+    private List<ulong> m_ClientIndices = new List<ulong>();
+
+    private bool   m_MutliParameterCanSend;
+
+    private int m_MutliParameterIntValue;
+    private int m_MutliParameterValuesCount;
+    private int m_MutliParameterNoneCount;
+    private int m_RpcMessagesSent;
+    private int m_GlobalCounter;
+    private int m_GlobalDirectCounter;
+    private int m_GlobalDirectCurrentClientIdIndex;
+    private int m_localClientCounter;
+    private int m_GlobalCounterOffset;
+    private int m_RpcPerSecond;
+    private int m_GlobalDirectScale;
+
+    private long  m_MutliParameterLongValue;
+    private ulong m_LocalClientId;
+
+    private float m_MutliParameterFloatValue;
+    private float m_GlobalCounterDelay;
+    private float m_DirectGlobalCounterDelay;
+    private float m_LocalCounterDelay;
+    private float m_LocalMultiDelay;
+    private float m_RpcPerSecondTimer;
+    private float m_GlobalDirectFrequency;
```

Comment:
```suggestion
    private bool   m_MultiParameterCanSend;

    private int m_MultiParameterIntValue;
    private int m_MultiParameterValuesCount;
    private int m_MultiParameterNoneCount;
    private int m_RpcMessagesSent;
    private int m_GlobalCounter;
    private int m_GlobalDirectCounter;
    private int m_GlobalDirectCurrentClientIdIndex;
    private int m_LocalClientCounter;
    private int m_GlobalCounterOffset;
    private int m_RpcPerSecond;
    private int m_GlobalDirectScale;

    private long  m_MultiParameterLongValue;
    private ulong m_LocalClientId;

    private float m_MultiParameterFloatValue;
    private float m_GlobalCounterDelay;
    private float m_DirectGlobalCounterDelay;
    private float m_LocalCounterDelay;
    private float m_LocalMultiDelay;
    private float m_RpcPerSecondTimer;
    private float m_GlobalDirectFrequency;
```

Reply:
I applied this update in b798e82 which included all references to said variables.